lombok.config

```diff
@@ -16,3 +16,5 @@
 #
 
 lombok.addLombokGeneratedAnnotation = true
```

Comment:
I prefer that we keep the same formatting of this file. You might either remove the blanks around `=` in this line or add blanks around `=` in next line.